samples/Mvc.Client/Mvc.Client.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Server.IISIntegration" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="$(AspNetCoreVersion)" />
```

Comment:
This should be using the .All package

Reply:
@davidfowl `dotnet restore` says `Microsoft.AspNetCore.All 2.0.0` does not support `net461`, the target framework this sample used.